rs/ledger_suite/icrc1/ledger/tests/tests.rs

```diff
@@ -1765,7 +1736,7 @@ mod incompatible_token_type_upgrade {
     }
 
     #[test]
-    #[should_panic(expected = "failed to decode ledger state")]
+    #[should_panic(expected = "invalid type: enum, expected u64 or { e8s: u64 }")]
```

Comment:
Why did this change?

Reply:
I think I used a wrong combination of mainnet canisters in `mainnet-canisters.json`. The canisters and the error message were updated in https://github.com/dfinity/ic/pull/2547 so I reverted my change to json and this error message.